stdlib/array.gr

```diff
@@ -373,3 +373,47 @@ export let unique = (array) => {
     array
   )
 }
+
+/**
+ * Slices an array given zero-based start and end indexes. The value
+ * at the end index will not be included in the result.
+ *
+ * If either index is a negative number, it will be treated as a reverse index from
+ * the end of the array. e.g. slice(1, -1, [> 'a', 'b', 'c']) == [> 'b'].
+ *
+ * @param stardIndex: Number The index of the array where slicing starts
+ * @param endIndex: Number The index of the array where slicing ends, but the element will not be included
```

Comment:
Two things:
1. Typo in `startIndex`
2. This might not be as spelled-out as the version you wrote, but I think it is more clear and precise.
```suggestion
 * @param startIndex: Number The index of the array where the slice will begin (inclusive)
 * @param endIndex: Number The index of the array where the slice will end (exclusive)
```

Reply:
I understood both, but the second is more precise.